packages/template-retail-react-app/translations/en-US.json

```diff
@@ -89,6 +89,12 @@
   "account_order_detail.title.order_details": {
     "defaultMessage": "Order Details"
   },
+  "account_order_detail.toast.cancellation_failed": {
+    "defaultMessage": "Request cancellation did not succeed."
```

Comment:
Is this the order being canceled?
Can the user do something about the cancellation failure it or is it a system failure? If the there is an action the user can take to fix the error, it usually should be part of the error message. 
If it is a system failure that the user can't do anything about, then we use this style of error message:
"Something went wrong with the order cancellation."